### ISSUE TITLE
npm package broken

### ISSUE THREAD

**Issue by emilos** [huncwotjs/huncwot#65]
The current version in npm seems to be broken:

The issue seems to be fixed in the meantime, do you mind publishing a new one?

It might be a good idea to add some basic local specs and set up a linter (like standard.js) to avoid issues like this.

I'd be happy to help if those ideas make sense to you.

```
/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/huncwot/response.js:139        ││/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/huncwot/response.js:139       │
│  Format,                                                                                  ││  Format,                                                                                 │
│  ^                                                                                        ││  ^                                                                                       │
│                                                                                           ││                                                                                          │
│ReferenceError: Format is not defined                                                      ││ReferenceError: Format is not defined                                                     │
│    at Object.<anonymous> (/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/hunc││    at Object.<anonymous> (/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/hun│
│wot/response.js:139:3)                                                                     ││cwot/response.js:139:3)                                                                   │
│    at Module._compile (internal/modules/cjs/loader.js:955:30)                             ││    at Module._compile (internal/modules/cjs/loader.js:955:30)                            │
│    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)               ││    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)              │
│    at Module.load (internal/modules/cjs/loader.js:811:32)                                 ││    at Module.load (internal/modules/cjs/loader.js:811:32)                                │
│    at Function.Module._load (internal/modules/cjs/loader.js:723:14)                       ││    at Function.Module._load (internal/modules/cjs/loader.js:723:14)                      │
│    at Module.require (internal/modules/cjs/loader.js:848:19)                              ││    at Module.require (internal/modules/cjs/loader.js:848:19)                             │
│    at require (internal/modules/cjs/helpers.js:74:18)                                     ││    at require (internal/modules/cjs/helpers.js:74:18)                                    │
│    at Object.<anonymous> (/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/hunc││    at Object.<anonymous> (/Users/emilos/Desktop/Projects/emilos/test-app/node_modules/hun│
│wot/index.js:27:22)                                                                        ││cwot/index.js:27:22)                                                                      │
│    at Module._compile (internal/modules/cjs/loader.js:955:30)                             ││    at Module._compile (internal/modules/cjs/loader.js:955:30)                            │
│    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)               ││    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)              │
│                                                                                           ││                                                                                          │
│ ..:: PROGRAM TERMINATED (code: 1) ::..                                                    ││ ..:: PROGRAM TERMINATED (code: 1) ::.
```

**Comment by zaiste** [comment]
@emilos I'm sorry for the trouble! I've just released a `0.55.0`. Would it solve the problem on your side? 

I absolutely agree with adding specs and linter. I'll dive into this tonight. By the way, we could have a call in the afternoon if you're available.

**Comment by emilos** [comment]
Thanks, works now! Having some other issues, but they seem unrelated to this one, so closing.
